app/_how-tos/configure-google-cloud-secret-as-a-vault-backend.md

```diff
@@ -29,23 +28,76 @@ tags:
     - secrets-management
 
 tldr:
-    q: How do I 
-    a: placeholder
+    q: How do I use Google Cloud Secret Manager as a Vault in {{site.base_gateway}}?
+    a: |
+      Save a secret in [Google Cloud Secret Manager](https://console.cloud.google.com/security/secret-manager) and create a service account with the `Secret Manager Secret Accessor` role. Export your service account key JSON as an environment variable (`GCP_SERVICE_ACCOUNT`), set `lua_ssl_trusted_certificate=system` in your `kong.conf` file, then configure a Vault entity with your Secret Manager configuration. Reference secrets from your Secret Manager vault like the following: `{vault://gcp-sm-vault/test-secret}`
 
 tools:
     - deck
 
+prereqs:
+  gateway:
+    - name: GCP_SERVICE_ACCOUNT
+    - name: KONG_LUA_SSL_TRUSTED_CERTIFICATE
+  inline:
+    - title: Google Cloud Secret Manager
+      position: before
+      content: |
+        To add Secret Manager as a Vault backend to {{site.base_gateway}}, you must configure the following:
+
+        1. In the [Google Cloud console](https://console.cloud.google.com/), create a project and name it `test-gateway-vault`.
+        1. On the [Secret Manager page](https://console.cloud.google.com/security/secret-manager), create a secret called `test-secret` with the following JSON content:
+            ```json
+            secret
+            ```
+        1. Create a service account key:
+            1. In the [Google Cloud console](https://console.cloud.google.com/), click the `test-gateway-vault` project.
+            1. Click the email address of the service account that you want to create a key for.
+            1. From the Keys tab, create a new key from the add key menu and select JSON for the key type.
+            1. Save the JSON file you downloaded.
+            1. From the [IAM page](https://console.cloud.google.com/iam-admin/iam?supportedpurview=project), grant access to the [`Secret Manager Secret Accessor` role for your service account](https://cloud.google.com/secret-manager/docs/access-secret-version#required_roles).
+      icon_url: /assets/icons/google-cloud.svg
+    - title: Environment variables
+      position: before
+      content: |
+          Set the environment variables needed to authenticate to Google Cloud:
+          ```sh
+          export GCP_SERVICE_ACCOUNT=$(cat /path/to/file/service-account.json)
+          export KONG_LUA_SSL_TRUSTED_CERTIFICATE='system'
+          ```
 
+          Note that the both variables need to be passed when creating your Data Plane container.
+      icon_url: /assets/icons/file.svg
 cleanup:
```

Comment:
Should we have a cleanup step for GCP resources?

Reply:
I've added one now!